roles/ceph-rgw-loadbalancer/defaults/main.yml

```diff
@@ -5,6 +5,10 @@
 # GENERAL #
 ###########
 
+keepalived_repository: distro
```

Comment:
The example we used in ceph-ansible doesn't apply quite as well here - their installation options included:
- `distro`
- `repository`

Our options here are:
- `distro`
- `download` (or `url` perhaps?)

Could rename the variable for more clarity: `keepalived_package_source:`

Reply:
done